src/sentry/mail/notifications.py

```diff
@@ -48,12 +54,19 @@ def get_unsubscribe_link(user_id: int, group_id: int) -> str:
     )
 
 
-def can_users_unsubscribe(notification: ActivityNotification) -> bool:
+def can_users_unsubscribe(notification: BaseNotification) -> bool:
     return bool(notification.group)
 
 
+def log_message(notification: BaseNotification, user: User) -> None:
+    pass
```

Comment:
is this meant to be fleshed out? it's called in `send_notification_as_email`

Reply:
This is just creating the hook so that I can use it in the next PR where I conditionally log if this is a `RuleNotification`.